docs/reference/setup/install/zip-windows-start.asciidoc

```diff
@@ -1,19 +1,35 @@
-==== Running Elasticsearch from the command line
+==== Run {es} from the command line
```

Comment:
Should we have a note in all of these pages to say that "if you intend this node to be part of an existing cluster, please look at <this page> where we describe the enrollment process ?

Reply:
We could link to the steps for [enrolling nodes in your cluster](https://www.elastic.co/guide/en/elasticsearch/reference/master/configuring-stack-security.html#stack-enroll-nodes), or pull those steps directly into these pages. Doing so saves users an extra click to navigate away from this page.